test/api_integration/apis/search/bsearch.ts

```diff
@@ -169,7 +169,7 @@ export default function ({ getService }: FtrProviderContext) {
           expect(resp.status).to.be(200);
           parseBfetchResponse(resp).forEach((responseJson, i) => {
             expect(responseJson.id).to.be(i);
-            verifyErrorResponse(responseJson.error, 400, 'search_phase_execution_exception', true);
+            verifyErrorResponse(responseJson.error, 400, 'search_phase_execution_exception', false);
```

Comment:
I'm not sure I understand why this no longer needs to check the body for attributes. Could you explain?

Reply:
I am not sure, but they are no longer there with the default distribution.